src/Kit/System/Watchdog/WatchedEventThread.cpp

```diff
@@ -0,0 +1,91 @@
+/*------------------------------------------------------------------------------
+ * Copyright Integer Fox Authors
+ *
+ * Distributed under the BSD 3 Clause License. See the license agreement at:
+ * https://github.com/Integerfox/kit.core/blob/main/LICENSE
+ *
+ * Redistributions of the source code must retain the above copyright notice.
+ *----------------------------------------------------------------------------*/
+/** @file */
+
+#include "Kit/System/Watchdog/WatchedEventThread.h"
```

Comment:
In .cpp and .c files - you do not need to specify header file paths - if the header file is co-located in the same directory as the .cpp/.c file.
```suggestion
#include "WatchedEventThread.h"
#include "Supervisor.h"
#include "Kit/System/ElapsedTime.h"
```